docs/DeveloperGuide.md

```diff
@@ -255,31 +255,58 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### Product scope
 
-**Target user profile**:
+**Target user:** [School of Computing](https://www.comp.nus.edu.sg) (SoC) students who:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* Need to manage large number of contacts and events
+* Prefer desktop apps over other types
+* Can type fast and prefer typing to mouse interactions
+* Are reasonably comfortable using CLI apps
 
+**Value proposition**: Manage contacts of peers, professors and events such as classes and Co-Curricular Activities in
+a single system faster than a typical mouse/GUI driven app while maintaining user-friendliness
```

Comment:
should we have space after "/"?

Reply:
The original develop guide does not leave space after "/", so should we just stick to this format? Or do you think having a space after "/" is better? What do you think @Gordon25?

Reply:
I think we can follow the original developer guide.